javascript/src/pre-recorded/pre-recorded-url.js

```diff
@@ -1,4 +1,29 @@
-import axios, { AxiosError } from "axios";
```

Comment:
Why the change ?  It looks terrible.
If the `"type": "module"` is a problem, make a change to generate .mjs file maybe ?